bool.go

```diff
@@ -0,0 +1,45 @@
+package nulltypes
+
+import (
+	"database/sql/driver"
+	"encoding/json"
+)
+
+//NullBool is a wrapper around bool
+type NullBool struct {
+	Bool  bool
+	Valid bool
+}
+
+// Bool method to get NullBool object from bool
+func Bool(Bool bool) NullBool {
+	return NullBool{Bool, true}
```

Comment:
It can be a named assignment

```
NullBool{x: _, y: _}
```

Reply:
It can be. But no need to put extra code here. We should make it even shorter than that if we can. 😉